polykit/analysis/contact_maps.py

```diff
@@ -585,3 +587,51 @@ def monomerResolutionContactMapSubchains(
         uniqueContacts=True,
         nproc=n,
     )
+
+
+def coolify(data,
```

Comment:
would renaming `data` --> `contact_matrix` be more clear?

some other places in polychrom/polykit data is an Nx3 array of xyz positions... (though of course that should probably be renamed `positions`....)